solutions/communication/index.asciidoc

```diff
@@ -0,0 +1,108 @@
+//Category=Communication
+//Products=Kafka; Istio
+//Maturity level=Initial
+
+= Communication
+
+== Communication among services
+Communication is a very important aspect while building IT application, especially large and complex one or one with microservices architecture. Based on your analysis of your problems, considering the following basic questions to start with:
+
+* Which services have to communicate with which services? to exchange which type of data?
+* How should the communication look like? in a synchronous way or asynchronous way?
+
+Once you have your answers, make your choices on concrete protocols, products and libraries. Using multiple of them in your application is fine.
+
+Unfortunately, there is no silver bullet. Each communication type has its own advantages and disadvantages and target a different scenario and goals. With that said, do not always apply REST or always apply kafka just because you have heard some success stories about it.
+
+Here we try to expose cross-cutting best practices for communication among services in larger IT application landscapes on a high level. Details how to implement this with specific libraries or programming-languages are described in the individual stacks of devonfw.
+
+== Communication types
+Generally, types of communication can be classified in 2 axes.
+
+The first axis defines if the protocol is synchronous or asynchronous:
```

Comment:
Should we add an image here?

Reply:
I am not sure which kind of image can fit here. The idea is quite simple and is explained well in the text. A slightly unprecise image can confuse the reader.

Reply:
Maybe a sequence diagram

Reply:
image added